packages/back-end/src/services/organizations.ts

```diff
@@ -1149,6 +1149,7 @@ export function getSubscriptionFromOrg(
       trialEnd: organization.subscription.trialEnd,
       status: getStripeSubscriptionStatus(organization.subscription.status),
       hasPaymentMethod: !!organization.subscription.hasPaymentMethod,
+      hasLicense: false,
```

Comment:
This is used to determine if we have the information needed to fetch the customer's cards from Stripe and display them in the app.